src/components/Country.js

```diff
@@ -0,0 +1,9 @@
+const Country = (countryObj) => {
+  const title = document.createElement("h2");
+  title.className = "country-item__title";
+  title.textContent = countryObj.name.common;
```

Comment:
un chan operator seria bueno  aqui o  o se si debemos poner un texto por defautl (N/A) (No Name) 🤔 la idea es evitar que la aplicacion treune si esto falla

 ```suggestion
  title.textContent = countryObj?.name?.common;
```